apps/cowswap-frontend/src/modules/bridge/pure/contents/QuoteBridgeContent/index.tsx

```diff
@@ -67,8 +69,8 @@ export function QuoteBridgeContent({
 
   return (
     <>
-      {contents.filter(isTruthy).map(({ withTimelineDot, label, content }) => (
-        <ConfirmDetailsItem withTimelineDot={withTimelineDot} label={label}>
+      {contents.filter(isTruthy).map(({ withTimelineDot, label, content }, index) => (
+        <ConfirmDetailsItem key={index} withTimelineDot={withTimelineDot} label={label}>
           {content}
         </ConfirmDetailsItem>
       ))}
```

Comment:
Should we display the `max bridge slippage` here as well? It's what I had in my mockup versions.